.github/workflows/swift.yaml

```diff
@@ -30,6 +30,7 @@ on:
       - 'WorkflowUI/**'
       - '.github/workflows/swift.yaml'
       - '.buildscript/build_swift_docs.sh'
+      - '*.swift'
```

Comment:
What does this catch that the other globs didn't?

Reply:
Changes to `WorkflowReactiveSwift` or `WorkflowRxSwift` files were not caught.

We will need to re-evaluate this, it might be fine to run it on every PR, now that Kotlin is in it's own repo.